ra/ra.go

```diff
@@ -1140,29 +1110,7 @@ func (ra *RegistrationAuthorityImpl) FinalizeOrder(ctx context.Context, req *rap
 
 // NewCertificate requests the issuance of a certificate for the v1 flow.
 func (ra *RegistrationAuthorityImpl) NewCertificate(ctx context.Context, req *rapb.NewCertificateRequest) (*corepb.Certificate, error) {
-	// Verify request
-	if req == nil || req.Csr == nil || req.RegID == 0 {
-		return nil, errIncompleteGRPCRequest
-	}
-	// Deserialize csr
-	csr, err := x509.ParseCertificateRequest(req.Csr)
-	if err != nil {
-		return nil, err
-	}
-
-	// Verify the CSR
-	if err := csrlib.VerifyCSR(ctx, csr, ra.maxNames, &ra.keyPolicy, ra.PA, req.RegID); err != nil {
-		return nil, berrors.MalformedError(err.Error())
-	}
-	// NewCertificate provides an order ID of 0, indicating this is a classic ACME
-	// v1 issuance request from the new certificate endpoint that is not
-	// associated with an ACME v2 order.
-	cert, err := ra.issueCertificate(ctx, core.CertificateRequest{CSR: csr, Bytes: req.Csr},
-		accountID(req.RegID), orderID(0), issuance.IssuerNameID(req.IssuerNameID))
-	if err != nil {
-		return nil, err
-	}
-	return bgrpc.CertToPB(cert), nil
+	return nil, fmt.Errorf("The ACME v1 NewCertificate flow is deprecated")
```

Comment:
Just checked and `errors` is already imported so you can just GitHub commit this suggestion.
```suggestion
	return nil, errors.New("The ACME v1 NewCertificate flow is deprecated")
```